src/myuw-notifications.html

```diff
@@ -160,5 +159,5 @@ <h1>Notifications</h1>
         <li id="empty-state">
```

Comment:
In the empty state case, does this end up being an `<li>` outside of an `<ol>` or `<ul>`?

Reply:
I missed that! Changed to a <div>.